docs/CONFIG-PROPERTIES.md

```diff
@@ -72,6 +72,7 @@
 | log.dedup.window.size | integer | 0 | The size of the log deduplicator's sliding window (in number of messages). See logging [docs](LOGGING.md#log-filtering-counting-and-deduplication) for details. If the window size is set to 0 (default), no deduplication is performed. |
 | log.count.filenames | string | "" | Comma-separated list of log's filenames to be counted and logged once instead of logging them every time. Example `/my-pkg/main.go:123,/other-pkg/code.go:42`. Empty string `""` doesn't filter anything out, however a single comma `","` will filter out all entries that don't have a filename field set (e.g. logs not coming from components written in Golang). See logging [docs](LOGGING.md#log-filtering-counting-and-deduplication) for details. |
 | log.filter.filenames | string | "" | Comma-separated list of log's filenames to be filtered out. Example `/my-pkg/main.go:123,/other-pkg/code.go:42`. Empty string `""` doesn't filter anything out, however a single comma `","` will filter out all entries that don't have a filename field set (e.g. logs not coming from components written in Golang). See logging [docs](LOGGING.md#log-filtering-counting-and-deduplication) for details. |
+| edgeview.authen.publickey | string | "" | Specifies SSH RSA public keys used for Edgeview client command authentication. The user issuing an Edgeview command must sign it with their SSH private key, and the device verifies the signature using one of the public keys provided in this configuration. Multiple public keys should be separated by newline characters. |
```

Comment:
It might be difficult to separate them with newlines on the CLI and I think it's also more error prone. Should we maybe use semicolons instead?

Reply:
all the normally used files such as 'authorized_keys' or 'known_hosts' are using the newlines, the other character may be have potential issues. I actually found from 'zcli', just cut&paste the one, the hit the 'return' key, then keep cut&paste next one, it works fine.

Reply:
I'm afraid that the user is gonna try some unnecessary characters like spaces, backslashes etc and then will forget that they added those and will not know what's wrong...

Reply:
True. But no matter whatever delimiter we use, there will be mistakes sometimes. and i believe most of the time, people will use just one.

Reply:
When I read "The user issuing an Edgeview command must sign it" I first assumed "it" refered to the edgeview.authen.publickey, but I suspect "it" refers to "the command". Can you clarify this in the markdown text?

Reply:
You are right, this was not very clear. Modified the help text.